src/Clapi/Valuespace.hs

```diff
@@ -353,6 +370,10 @@ validateChildKeyTypes getType' np orderedKeys expectedTypes =
         (traverse . traverse) getType' expectedTypeMap
     failTypes taggedBadTypes
 
+childTypesOf :: Definition -> Path.Name -> Maybe TypePath
```

Comment:
This doesn't really seem like `childTypesOf`, more like `childTypeFor`. Although I see that perhaps that name is used somewhere else?